src/components/theme/Icon/Icon.jsx

```diff
@@ -17,6 +17,17 @@ const defaultSize = '36px';
  * @param {string} props.className className to add to the component.
  * @param {string} props.title Title (a11y).
  * @returns {string} Markup of the component.
+ *
+ * Use:
+ * drop icon to the icons folder ("src/icons")
+ * import svg into the file
+ * import this Icon component
+ * add icon component with name = your imported svg
+ *
+ * Reasoning:
+ * add a11y title to SVGs
```

Comment:
I would add https://kitconcept.com/blog/pastanaga-icon-system/
as reference.

Reply:
And also mention briefly that inlining an SVG is the best:
-  Zero conf Inlined SVGs, as it is the best option when working with SVGs